stglib/sg/wvsnc2waves.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from ..core import qaqc, utils, waves
+
+
+def nc_to_waves(nc_filename):
+    """
+    Process burst data to wave statistics
+    """
+
+    ds = xr.open_dataset(nc_filename)
+
+    # Check to see if need to make smaller wave bursts from really long wave bursts
+    if "calculated_wave_interval" in ds.attrs:
```

Comment:
In other instruments I believe we use "wave_interval"... is there a reason for the "calculated_" part?

Reply:
Yes, we are reporting the wave interval that was programmed into the sensor for deployment as "wave_interval".  But for processing, we want to calculate the wave statistics on a smaller wave interval than what was programmed.  So I added a "calculated_wave_interval" to the config file where we can specify a smaller wave interval.  The code will then subdivide the programmed wave bursts into the smaller wave bursts that we want for calculations.  I can change the name to something else if you want to call it something different.